integrationtests/test_login.py

```diff
@@ -19,11 +39,11 @@ def tearDownClass(cls):
     def test_login(self):
         b = self.browser
         b.visit(self.live_server_url + '/account/login')
-        b.fill('email', self.user.email)
+        b.fill('login', self.user.email)
         b.fill('password', self.password)
 
         b.find_by_tag('button').click()
-
+        time.sleep(2)
```

Comment:
why does login now require a sleep when it did not before? Is it substantially slower?

Reply:
I couldn't get this test to run locally without some kind of pause.